analysis/NECRvsLength.py

```diff
@@ -0,0 +1,145 @@
+import SiemensQuadraProperties as sqp
```

Comment:
What is the purpose of this file?

Reply:
It is used for code development so I moved it to another directory as it basically does the same thing as the corresponding jupyter notebook